cylc/flow/config.py

```diff
@@ -1513,6 +1512,19 @@ def adopt_orphans(self, orphans):
             self.runtime['linearized ancestors'][orphan] = [orphan, 'root']
 
     def configure_workflow_state_polling_tasks(self):
+
+        # Deprecation warning - automatic workflow state polling tasks don't
+        # necessarily have deprecated config items outside the graph string.
+        if (
```

Comment:
IMO this should work for Cylc VR and VIP too.

Reply:
I'm bailing out for the night, but if the fix is as small as it seems (on your side-branch) I don't mind if you either:
- merge it to this branch yourself (after testing it)
- or re-raise it against master

... especially if it means this PR can get merged ASAP (for 8.3.0...)

Reply:
I can't merge it.

Reply:
Huh, I thought you should be able to (possibly after "reviewing" it yourself. Anyhow, I'll do it.